tests/urlHash.js

```diff
@@ -31,17 +31,40 @@ requirejs([
 
         it("should change the hash when you add a question", function() {
             util.loadXML("");
-            util.addQuestion("Text", "text");
-            assert.equal(window.location.hash, '#form/text');
+            util.addQuestion("Text", "initial");
+            util.clickQuestion("initial");
+            assert.equal(window.location.hash, "#form/initial");
         });
 
         it("should change the hash when you delete a question", function() {
             util.loadXML("");
+            
+            util.addQuestion("Text", "first");
+            util.clickQuestion("first");
+
+            util.addQuestion("Text", "second");
+            util.clickQuestion("second");
+            
+            assert.equal(window.location.hash, "#form/second");
+            
+            util.deleteQuestion("/data/second");
+            
+            assert.equal(window.location.hash, "#form/first");
+        });
+
+        it("should change the hash when you delete a question", function(done) {
```

Comment:
Did you intend to add a new test with the same name/description as the one above it?